bindata/network/ovn-kubernetes/common/udn-admission-policy.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: disable-udn-label-change
```

Comment:
Can we make it more user friendly? something like:
`user-defined-networks-namespace-label`